static/scss/search-page.scss

```diff
@@ -314,6 +314,31 @@
         padding-right: 5px;
       }
     }
+
+    .sorting-header {
+      margin: 0;
+
+      .sorting-row {
+        width: 100%;
+        padding: 20px 0;
+        text-transform: uppercase;
+        font-size: 12px;
+        color: $font-gray;
+
+        .selected {
+          color: black;
+        }
+
+        >div {
```

Comment:
effectively this just makes the whole row show a pointer cursor. if we want that, would it be better to put the style definition on `.sorting-row`? if not, i think maybe selecting for `.mdl-cell` would be clearer and more helpful here rather than `>div`. i had to go into chrome dev tools to figure out what mdl renders as a div in order to figure out the reasoning behind this selector. other developers will have to do that too